### ISSUE TITLE
arteriolar blood pressure

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3127]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal renal arteriolar blood pressure regulation
2. abnormal renal arteriolar blood pressure
3. increased efferent renal arteriolar blood pressure
4. decreased efferent renal arteriolar blood pressure

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. anomaly in the process that modulates the force with which blood travels through the renal circulatory system
2. anomaly in the tension of the blood within the renal arteries
3. increased tension of the blood within the renal arteries
4. decreased tension of the blood within the renal arteries
PMID:10070148

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.
>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal blood pressure regulation (MP:0010695)
2,  abnormal renal arteriolar blood pressure regulation (new term)
3.  abnormal renal arteriolar blood pressure (new term)
4.  abnormal renal arteriolar blood pressure (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjfl

**Comment by sbello** [comment]
Renal blood flow ref - https://www.ncbi.nlm.nih.gov/books/NBK482248/ 
Regulation of renal blood flow https://courses.lumenlearning.com/suny-ap2/chapter/regulation-of-renal-blood-flow/ 

There is a problem with the requested definition, the term name specifies arteriolar but the definitions are using arterial. Looking at the reference it appears that the definition should change to be arterioles rather than arteries.
